src/objects/npe01__Contacts_And_Orgs_Settings__c.object

```diff
@@ -95,6 +95,18 @@
         <type>Number</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Multiple_Tributes__c</fullName>
+        <description>Allows multiple Tributes per Opportunity.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>Specify that your org can have one tribute per Opportunity (single), one per Opportunity and multiple tributes in separate Tributes (mixed), or multiple tributes in separate Tribute records. Choosing multiple creates Tributes only for new Opportunities.</inlineHelpText>
+        <label>Multiple Tributes</label>
+        <length>255</length>
```

Comment:
Was this settled on as a single string word - and one that is not defined in a custom label. If so, I think the field length should be around 10. Also, how is null handled by the code and should that be referenced in the description (not inline help text since that would be visible to a user)?